### PR TITLE
Appstream

### PR DESCRIPTION
**Pull request by sergiusens**
- [ ] Have you followed the [guidelines for contributing](https://github.com/snapcore/snapcraft/blob/master/CONTRIBUTING.md)?
- [ ] Have you signed the [CLA](http://www.ubuntu.com/legal/contributors/)?
- [ ] Have you successfully run `./runtests.sh static`?
- [ ] Have you successfully run `./runtests.sh tests/unit`?

-----

### CODE REVIEW

**Comment by apachelogger** [comment]
Wouldn't `<em>` typically be rendered as italic and wouldn't then `*` (i.e. italics) be a better translation to markdown? 

Other than that LGTM

**Comment by sergiusens** [comment]
Thanks @apachelogger, it is a very strict subset of markdown https://forum.snapcraft.io/t/use-of-markdown-in-snap-metadata-summary-description/2128/11

FWIW, `_` is also rendered as italics with markdown.

**Comment by apachelogger** [comment]
Fair enough

File: snapcraft/extractors/appstream.py [hunk 1]
```diff
@@ -138,7 +140,8 @@ def _get_xslt():
 def _get_value_from_xml_element(tree, key) -> Optional[str]:
     node = tree.find(key)
     if node is not None and node.text:
-        return node.text.strip()
+        # TODO: do this in the xslt
```
Comment (cjp256) [inline snapcraft/extractors/appstream.py]: Maybe ditch this comment if not going to happen (and perhaps just add why it's stripping all lines)

Comment (sergiusens) [reply snapcraft/extractors/appstream.py]: It is, I just need to learn how :-)

Comment (sergiusens) [reply snapcraft/extractors/appstream.py]: Added a extended comment

**Pull request merged**